tests/translator/input/graphqlapi_multiple_none_datasource_functions.yaml

```diff
@@ -0,0 +1,37 @@
+Transform: AWS::Serverless-2016-10-31
+Resources:
+  SuperCoolAPI:
+    Type: AWS::Serverless::GraphQLApi
+    Properties:
+      Name: SomeApi
+      SchemaInline: |
+        type Mutation {
+          addTodo(id: ID!, name: String, description: String, priority: Int): Todo
+        }
+      Auth:
+        Type: AWS_IAM
+      Functions:
+        MyFunction:
+          CodeUri: my-code-uri
+          DataSource: NONE
+          Runtime:
+            Name: some-runtime
+            Version: 1.2.3
+        AnotherFunction:
+          CodeUri: my-code-uri
+          DataSource: None
+          Runtime:
+            Name: some-runtime
+            Version: 1.2.3
```

Comment:
What's the expected output if I have 2 `none` data sources but with different `CodeUri`s?

Reply:
The `CodeUri` property does not have anything to do with how the `none` datasource is created. The output would be the same as this, except the transformed function would have a different `CodeS3Location` value.

Reply:
So one of the `CodeUri`s would be silently ignored from output?

Reply:
The output would be like this

```
<ApiName>NoneDataSource:
  Type: AWS::AppSync::DataSource
  Properties:
    Type: NONE
    ...
    
<ApiName>MyFunction:
  Type: AWS::AppSync::FunctionConfiguration
  Properties:
    CodeS3Location: my-code-uri
    DataSourceName: <ApiName>NoneDataSource
    ...

<ApiName>AnotherFunction:
  Type: AWS::AppSync::FunctionConfiguration
  Properties:
    CodeS3Location: another-code-uri
    DataSourceName: <ApiName>NoneDataSource
    ...
```